app/controllers/api/v1/activities_controller.rb

```diff
@@ -6,6 +6,9 @@ def latest
 
   def just_updated
     versions = Version.just_updated(50)
+    if params[:since].present?
```

Comment:
We also need to limit this. We don't want to break the database or app server with a huge request.